src/shared/lib/validation/user.ts

```diff
@@ -20,3 +22,12 @@ export function validateUserStatus(status: string): Validation<UserStatus> {
   const userStatus = parseUserStatus(status);
   return userStatus ? valid(userStatus) : invalid(['Invalid user status specified']);
 }
+
+export function validateCapability(raw: string): Validation<string> {
+  return SKILLS.includes(raw) ? valid(raw) : invalid(['Invalid capability specified.']);
+}
+
+export function validateCapabilities(raw: string[]): ArrayValidation<string> {
+  const validatedArray = validateArray(raw, validateCapability);
+  return mapValid(validatedArray, v => uniq(v));
+}
```

Comment:
This can be moved to `shared/lib/validation/index.ts` and re-exported from this module and the SWU opp validation module.